railties/test/isolation/abstract_unit.rb

```diff
@@ -603,7 +603,7 @@ def self.sh(cmd)
   FileUtils.rm_rf(app_template_path)
   FileUtils.mkdir_p(app_template_path)
 
-  sh "#{Gem.ruby} #{RAILS_FRAMEWORK_ROOT}/railties/exe/rails new #{app_template_path} --skip-bundle --no-rc --quiet"
+  sh "#{Gem.ruby} #{RAILS_FRAMEWORK_ROOT}/railties/exe/rails new #{app_template_path} --asset-pipeline=sprockets --skip-bundle --no-rc --quiet"
```

Comment:
Isn't this just delaying the problem? The test app is still generating using sprockets. Who is supposed to fix this?